redirector.k8s.io/deployment.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redirector-k8s-io
+  labels:
+    app: redirector-k8s-io
+    version: dirty
+spec:
+  revisionHistoryLimit: 2
+  replicas: 2
+  strategy:
+    rollingUpdate:
+      maxSurge: 4
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      app: redirector-k8s-io
+      component: txtdirect
+  template:
+    metadata:
+      labels:
+        app: redirector-k8s-io
+        component: txtdirect
+    spec:
+      securityContext:
+        fsGroup: 1337
```

Comment:
why?  comment

Reply:
Generally used to make sure files are owned correctly. Used in other deployments that we have.
If this feels unnecessary, happy to remove.

Reply:
I meant why this number -- I know what the feature does :)

Reply:
Thought you wanted to get the reasoning behind why the feature was used ;)
Basically just a random number that wasn't used for something specific yet and was easy to remember.